### PR TITLE
feat: getSupportedAssets and tests added

### LINKED ISSUE #4250: Developers can display available assets to end users
**Problem**
As an end user, I should only be allowed to use available assets (from UI + connext/uni infra).

**Solution**
    - Most applications have a hardcoded list of assets they support. We can rely on them to limit to assets they support.
        - For chain abstracted workflows, we can add SDK method like `getSupportedAssets`using whatever is available in Uniswap

### PR DESCRIPTION
**Pull request by prathmeshkhandelwal1**
## Description

Added methods for utility of getting all tokens for a specific chain. 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

Fixes #4250

### CODE REVIEW

File: packages/agents/chain-abstraction/test/helpers/asset.spec.ts [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    beforeEach(() => {
+      axiosGetStub = stub(MockableFns, "axiosGet");
+    });
+    afterEach(() => {
+      restore();
+      reset();
+    });
+    it("Should work with getting supported asset", async () => {
+      const mockChainID = 1;
+      axiosGetStub.resolves({ data: { tokens: mockUniswapResponse } });
```
Comment (rhlsthrm) [inline packages/agents/chain-abstraction/test/helpers/asset.spec.ts:78]: This mock should work

File: packages/agents/chain-abstraction/src/helpers/asset.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
+import { jsonifyError } from "@connext/nxtp-utils";
+import { axiosGet } from "@connext/nxtp-utils";
```
Comment (rhlsthrm) [inline packages/agents/chain-abstraction/src/helpers/asset.ts:2]: This needs to be imported from mockable functions in order to make the mocks work.

**Review by rhlsthrm — CHANGES_REQUESTED** [review]

File: packages/agents/chain-abstraction/test/helpers/asset.spec.ts (hunk 1 above)
Comment (wanglonghong) [reply packages/agents/chain-abstraction/test/helpers/asset.spec.ts:78]: https://github.com/connext/monorepo/pull/4324/commits/e4bfb51d8109879f2afe3c8926966f0c8f3a92a4

File: packages/agents/chain-abstraction/src/helpers/asset.ts (hunk 2 above)
Comment (wanglonghong) [reply packages/agents/chain-abstraction/src/helpers/asset.ts]: https://github.com/connext/monorepo/pull/4324/commits/e4bfb51d8109879f2afe3c8926966f0c8f3a92a4

**Comment by prathmeshkhandelwal1** [comment]
Thanks @wanglonghong

**Review by wanglonghong — APPROVED** [review]

**Review by wanglonghong — APPROVED** [review]

**Review by rhlsthrm — APPROVED** [review]

**Pull request merged**